# HG changeset patch
# User Rik <rik@octave.org>
# Date 1423426205 28800
#      Sun Feb 08 12:10:05 2015 -0800
# Node ID 890ff06d84ce852f97effe49b06e4e00cbc8beaf
# Parent  f1270e5a31178f5a34fe836b0c3c17cee35cd71b
record.m: Remove seealso link to @audiorecorder which stops docs building.

* record.m: Remove seealso link to @audiorecorder which stops docs building.

diff --git a/scripts/audio/record.m b/scripts/audio/record.m
--- a/scripts/audio/record.m
+++ b/scripts/audio/record.m
@@ -20,17 +20,16 @@
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} record (@var{sec})
 ## @deftypefnx {Function File} {} record (@var{sec}, @var{fs})
 ## Record @var{sec} seconds of audio from the system's default audio input at
 ## a sampling rate of 8000 samples per second.  If the optional argument
 ## @var{fs} is given, it specifies the sampling rate for recording.
 ##
 ## For more control over audio recording, use the @code{audiorecorder} class.
-## @seealso{@audiorecorder/audiorecorder}
 ## @end deftypefn
 
 function x = record (sec, fs)
 
   if (nargin == 1)
     fs = 8000;
   elseif (nargin != 2)
     print_usage ();
